go/cmd/vtcombo/tablet_map.go

```diff
@@ -223,7 +223,10 @@ func (itc *internalTabletConn) StreamExecute(ctx context.Context, query string,
 	}()
 
 	return result, func() error {
-		return tabletconn.TabletErrorFromGRPC(tabletserver.ToGRPCError(finalErr))
+		if finalErr != nil {
+			return tabletconn.TabletErrorFromGRPC(tabletserver.ToGRPCError(finalErr))
```

Comment:
Since TabletErrorFromGRPC is a public function now, it should probably do a nil check.